Parsers/ASimNetworkSession/Parsers/vimNetworkSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,362 @@
+    Type: dynamic
+    Default: dynamic([])
+  - Name: ipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dstportnumber
+    Type: int
+    Default: int(null)
+  - Name: dvcaction
+    Type: dynamic
+    Default: dynamic([])
+  - Name: hostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+  [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let EventResultDvcActionLookup = datatable (
+      DeviceAction: string,
+      DvcAction: string,
+      EventResult: string
+  )
+      [
+      "allow", "Allow", "Success",
+      "deny", "Deny", "Failure",
+      "reset client", "Reset Source", "Failure",
+      "reset server", "Reset Destination", "Failure",
+      "reset both", "Reset", "Failure",
+      "drop", "Drop", "Failure",
+      "drop ICMP", "Drop ICMP", "Failure"
+  ];
+  let EventResultDetailsLookup = datatable(Reason: string, EventResultDetails: string)[
+      "threat", "Reset",
+      "policy-deny", "Unknown",
+      "decrypt-cert-validation", "Terminated",
+      "decrypt-unsupport-param", "Terminated",
+      "decrypt-error", "Terminated",
+      "tcp-rst-from-client", "Reset",
+      "tcp-rst-from-server", "Reset",
+      "resources-unavailable", "Unknown",
+      "tcp-fin", "Unknown",
+      "tcp-reuse", "Unknown",
+      "decoder", "Unknown",
+      "aged-out", "Unknown",
+      "unknown", "Unknown",
+      "n/a", "NA",
+  ];
+  let ThreatRiskLevelLookup = datatable(PanOSApplicationRisk: string, ThreatRiskLevel: int)
+  [
+      "1", 20,
+      "2", 40,
+      "3", 60,
+      "4", 80,
+      "5", 100
+  ];
+  let parser=(
+      disabled: bool=false, 
+      starttime: datetime=datetime(null), 
+      endtime: datetime=datetime(null), 
+      eventresult: string='*', 
+      srcipaddr_has_any_prefix: dynamic=dynamic([]),
+      dstipaddr_has_any_prefix: dynamic=dynamic([]),
+      ipaddr_has_any_prefix: dynamic=dynamic([]), 
+      hostname_has_any: dynamic=dynamic([]),
+      dstportnumber: int=int(null),
+      dvcaction: dynamic=dynamic([])
+      ) {
+      let src_or_any = set_union(srcipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+      let dst_or_any = set_union(dstipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+      CommonSecurityLog
+      | where not(disabled)
+          and (isnull(starttime) or TimeGenerated >= starttime)
+          and (isnull(endtime) or TimeGenerated <= endtime)
+          and DeviceVendor == "Palo Alto Networks" and DeviceProduct == "LF"
+          and DeviceEventClassID == "TRAFFIC"
+          and (eventresult == "*" or eventresult == "Success")
```

Comment:
@niralishah-crest @jayeshprajapaticrest 
Don't think this is correct, we are setting both success and failure. please validate

Reply:
I've updated filter to check with EventResult field